
resolution_x:   320
resolution_y:   240
epsilon:        5e-2
lambda:         0.1
control_rate:   1
resource_location:         "/home/cyh/Work/visual_servoing_ws/src/visual_servoing/franka_control_VS/param/"
image_rgb_desired_name:    "image_rgb_desired.png"
image_depth_desired_name:  "image_depth_desired.png"
image_rgb_initial_name:    "image_rgb_initial.png"
camera_intrinsic: [ 1.0, 0.0, 5.0, 
                    0.0, 1.0, 6.0, 
                    0.0, 0.0, 1.0]
pose_desired: [0.707107, -0.707107, -6.12323E-17, 0.306891,
-0.707107, -0.707107, -3.14018E-16, -2.36797E-16,
1.78747E-16, 2.65342E-16, -1.0, 0.590282,
0.0, 0.0, 0.0, 1.0]
joint_angle_initial: [0.0, -0.785398, 0.0, -2.35619, 0.0, 1.5708, 0.785398]
